wordpress-mobile/WordPress-iOS
# Small Reader Improvements: Add Related Posts

Part of #15942
WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/360

## Description
- Added related posts to the bottom of the reader detail screen 
- TODO: handle cell selection for related posts (Will make a separate PR)

## How to test
1. Go to reader tab
2. Tap on a reader post and open post details
3. Scroll to the bottom of the post
4. ✅ The related posts section should be displayed (Up to 2 "More from {blog}" posts / exactly 2 "More on WordPress.com" posts) 
5. ⚠️ Tapping the cells won't do anything for now

## Screenshot

Design ref: I5jpMcL84tYRh3FQJZs0fF-fi-315%3A89

![Simulator Screen Shot - iPhone 12 Pro - 2021-03-03 at 17 51 43](https://user-images.githubusercontent.com/6711616/109801316-a8c3ac80-7c61-11eb-97fa-6a022e8731af.png)| ![Simulator Screen Shot - iPhone 12 Pro - 2021-03-03 at 17 51 52](https://user-images.githubusercontent.com/6711616/109801322-a9f4d980-7c61-11eb-8e80-bfbcc10db90d.png)
--- | ---



## PR submission checklist:
